### PR TITLE
PACT-572 updated mock JSON data in the Web Application from 3 to 12 r…

### PR DESCRIPTION
**Pull request by userjd**
…ecords

### CODE REVIEW

File: test/controllers/EditSetControllerSpec.scala [hunk 1]
```diff
@@ -176,6 +176,24 @@ class EditSetControllerSpec extends BaseSpec {
                   "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (B)",
```
Comment (tna-erasmos) [inline test/controllers/EditSetControllerSpec.scala:176]: This should be reformatted (`sbt scalafmtAll`).

File: app/uk/gov/nationalarchives/omega/editorial/models/package.scala [hunk 2]
```diff
@@ -50,8 +50,8 @@ package object editSetRecords {
   val editSetRecord2: EditSetRecord = Json
```
Comment (tna-erasmos) [inline app/uk/gov/nationalarchives/omega/editorial/models/package.scala:50]: All of these `EditSetRecord` vals could be `private`.

**Review by tna-erasmos — COMMENTED** [review]
Minor points.

File: test/controllers/EditSetControllerSpec.scala (hunk 1 above)
Comment (tna-erasmos) [reply test/controllers/EditSetControllerSpec.scala:176]: The latest commit isn't formatted.

File: app/uk/gov/nationalarchives/omega/editorial/models/package.scala (hunk 2 above)
Comment (tna-erasmos) [reply app/uk/gov/nationalarchives/omega/editorial/models/package.scala]: Confirmed.

File: test/controllers/EditSetControllerSpec.scala (hunk 1 above)
Comment (userjd) [reply test/controllers/EditSetControllerSpec.scala:176]: @tna-erasmos please see last commit, it's reformatted

Comment (tna-erasmos) [reply test/controllers/EditSetControllerSpec.scala:176]: Confirmed.

**Review by tna-erasmos — APPROVED** [review]
LGTM

**Pull request merged**